server/routerlicious/packages/services/src/throttler.ts

```diff
@@ -51,7 +51,7 @@ export class Throttler implements IThrottler {
         // check cached throttle status, but allow operation through if status is not yet cached
         const cachedThrottlerResponse = this.throttlerResponseCache.get(id);
         if (cachedThrottlerResponse && cachedThrottlerResponse.throttleStatus) {
-            const retryAfterInSeconds = Math.ceil(cachedThrottlerResponse.retryAfterInMs / 1000);
+            const retryAfterInSeconds = cachedThrottlerResponse.retryAfterInMs;
```

Comment:
Please update the associated [ThrottlingError implementation and comment](https://github.com/microsoft/FluidFramework/blob/main/server/routerlicious/packages/services-core/src/throttler.ts#L29-L31) to reflect any naming/unit changes as well

Reply:
@tanviraumi reminded me that renaming the `retryAfter` property being thrown from the service would result in a breaking change between service and client, which is not ideal. I think leave the `retryAfter` name and unit (seconds) alone for now wherever it is being sent by the service and read by the client. This is can be converted to Ms in the r11s-driver's RestWrapper (and/or errorUtils.ts) so that everything above the driver still sees Ms

Reply:
ok! for the last sentence, do you mean it's ok to convert to Ms in r11s-driver's RestWrapper and errorUtils?